addons/bus/models/bus.py

```diff
@@ -154,6 +157,23 @@ def _bus_last_id(self):
         return last.id if last else 0
 
 
+class BusSendableMixin(models.AbstractModel):
+    _name = 'bus.sendable'
+    _description = 'Can send messages via bus.bus'
```

Comment:
The mixin should probably go with a comment that the object should also be handled some way in `_build_bus_channel_list` for the feature to work, just inheriting the mixin is not sufficient it's just there to ease the sending syntax.